sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -114,6 +114,7 @@ class Engine(val config: EngineConfig) {
       compiledPackages = compiledPackages,
       loadPackage = loadPackage,
       forbidLocalContractIds = config.forbidLocalContractIds,
+      costModel = preprocessing.CostModel.EmptyCostModelImplicits,
```

Comment:
Presumably, at a latter point in time, this will be derived from engine configuration?